src/game_api/entities_items.hpp

```diff
@@ -36,7 +36,7 @@ class Bomb : public Movable
     size_t unknown1;
     float scale_hor; // 1.25 = default regular bomb ; 1.875 = default giant bomb ; > 1.25 generates ENT_TYPE_FX_POWEREDEXPLOSION
     float scale_ver;
-    bool unknown2;
+    bool is_big_bomb; // is bomb from powerpack
```

Comment:
If you add new comments, try placing them using the usual `///` in front of the member so the docs pick it up.

Reply:
oh, that's a thing? maybe i should edit more of the comments that way?

Reply:
I think it should work. And yes, they should be in front.
If you want to you are free to change more of them, but there's no need to do that right now, so you can focus on something you enjoy more  :)